game/templates/game/stats.html

```
@@ -65,5 +81,37 @@ <h2>Recent Games</h2>
     </div>
     {% endif %}
 
+    <div id="gameSummaryOverlay" id="gameSummaryOverlay" role="dialog" aria-modal="true" aria-labelledby="summaryTitle" 
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ id ] was found.

(attr-no-duplication)